### ISSUE TITLE
docker build failed

### ISSUE THREAD

**Issue by infofromca** [OrchardCMS/OrchardCore#15648]
### To Reproduce
1. Go to 'the root folder of cms.web'
2. docker build --build-arg  TARGETOS=linux -t linuxoc  .

3. See error
/app/OrchardCore/Directory.Build.props(3,3): error MSB4019: The imported project "/Directory.Build.props" was not found. Confirm that the expression in the Import declaration "../../Directory.Build.props" is correct, and that the file exists on disk. [/app/OrchardCore/OrchardCore.Application.Cms.Targets/OrchardCore.Application.Cms.Targets.csproj]
24.28 /app/OrchardCore/Directory.Build.props(3,3): error MSB4019: The imported project "/Directory.Build.props" was not found. Confirm that the expression in the Import declaration "../../Directory.Build.props" is correct, and that the file exists on disk. [/app/OrchardCore/OrchardCore.AuditTrail.Abstractions/OrchardCore.AuditTrail.Abstractions.csproj]
24.29 /app/OrchardCore.Modules/Directory.Build.props(3,3): error MSB4019: The imported project "/Directory.Build.props" was not found. Confirm that the expression in the Import declaration "../../Directory.Build.props" is correct, and that the file exists on disk. [/app/OrchardCore.Modules/OrchardCore.Autoroute/OrchardCore.Autoroute.csproj]
24.29 /app/OrchardCore.Modules/Directory.Build.props(3,3): error MSB4019: The imported project "/Directory.Build.props" was not found. Confirm that the expression in the Import declaration "../../Directory.Build.props" is correct, and that the file exists on disk. [/app/OrchardCore.Modules/OrchardCore.Alias/OrchardCore.Alias.csproj]
24.29 /app/OrchardCore/Directory.Build.props(3,3): error MSB4019: The imported project "/Directory.Build.props" was not found. Confirm that the expression in the Import declaration "../../Directory.Build.props" is correct, and that the file exists on disk. [/app/OrchardCore/OrchardCore.Admin.Abstractions/OrchardCore.Admin.Abstractions.csproj]
### Expected behavior

4 because file Directory.Build.props under first OrchardCore is outside .src  which could not be copied  to /app
by this COPY ./src /app

**Comment by Piedone** [comment]
Apart from publishing to Docker Hub, what does the [preview build](https://github.com/OrchardCMS/OrchardCore/actions/runs/8533215657/workflow#L79) do differently that it works?

**Comment by infofromca** [comment]
did the preview use dockerfile

**Comment by Piedone** [comment]
Yes: https://github.com/OrchardCMS/OrchardCore/blob/main/Dockerfile-CI

**Comment by infofromca** [comment]
CLICK  the link of -CI, I could not find dockerfile which COPY ./src /app on line 9

**Comment by sebastienros** [comment]
I confirm the Dockerfile is broken because Directory.Build.props is not imported. The `COPY ./src /app` should be changed/

**Comment by sebastienros** [comment]
For @Piedone the preview in CI is actually copying the already built folder that the CI produced. We have two docker files in the root folder.
